subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -570,6 +574,69 @@ struct pdu_data_llctrl_min_used_chans_ind {
 	uint8_t min_used_chans;
 } __packed;
 
+struct pdu_data_llctrl_cis_req {
+	uint8_t cig_id;
+	uint8_t cis_id;
+	uint8_t c_phy;
+	uint8_t p_phy;
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+	uint16_t c_max_sdu:12;
+	uint16_t rfu0:3;
+	uint16_t framed:1;
+	uint16_t p_max_sdu:12;
+	uint16_t rfu1:4;
+#elif __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
+	uint16_t framed:1;
+	uint16_t rfu0:3;
+	uint16_t c_max_sdu:12;
+	uint16_t rfu1:4;
+	uint16_t p_max_sdu:12;
+#else
+#error "Unsupported endianness"
+#endif
+	uint8_t  c_sdu_interval[3];
+	uint8_t  p_sdu_interval[3];
+	uint16_t c_max_pdu;
```

Comment:
For this and a couple of places below, you could do a `uint64_t` bitfield when you have 24 + 24 + 16 => 64 bits. If you prefer 3 byte arrays, that's just fine as well :)

Reply:
3-byte arrays seem to be the thing to use for 24-bit values in other places, so I'll keep them.